doc/source/howtos/virt_sandbox.rst

```diff
@@ -141,169 +102,259 @@ with application servers.
 
-If you did everything correctly,
-the statistics page for HAProxy should look like this:
+     mysql> SELECT * FROM monitor.mysql_server_ping_log ORDER BY time_start_us DESC LIMIT 6;
+     +---------------+------+------------------+-------------------+------------+
+     | hostname      | port | time_start_us    | ping_success_time | ping_error |
+     +---------------+------+------------------+-------------------+------------+
+     | 192.168.70.61 | 3306 | 1469635762416190 | 948               | NULL       |
+     | 192.168.70.62 | 3306 | 1469635762416190 | 803               | NULL       |
+     | 192.168.70.63 | 3306 | 1469635762416190 | 711               | NULL       |
+     | 192.168.70.61 | 3306 | 1469635702416062 | 783               | NULL       |
+     | 192.168.70.62 | 3306 | 1469635702416062 | 631               | NULL       |
+     | 192.168.70.63 | 3306 | 1469635702416062 | 542               | NULL       |
+     +---------------+------+------------------+-------------------+------------+
+     6 rows in set (0.00 sec)
 
-.. image:: ../_static/pxc_haproxy_status_example.png
+  The previous examples show that ProxySQL is able to connect and ping the nodes
+  you added.
+
+  To enable monitoring of these nodes, load them at runtime:
+
+  .. code-block:: text
+
+     mysql> LOAD MYSQL SERVERS TO RUNTIME;
+
+  .. _proxysql-client-user:
+
+  .. rubric:: Creating ProxySQL Client User
+
+  ProxySQL must have users that can access backend nodes
+  to manage connections.
+
+  To add a user, insert credentials into ``mysql_users`` table:
+
+  .. code-block:: text
+
+     mysql> INSERT INTO mysql_users (username,password) VALUES ('sbuser','sbpass');
+     Query OK, 1 row affected (0.00 sec)
+
+  .. note::
+
+     ProxySQL currently doesn't encrypt passwords.
+
+  Load the user into runtime space and save these changes to disk
+  (ensuring that they persist after ProxySQL shuts down):
+
+  .. code-block:: text
+
+     mysql> LOAD MYSQL USERS TO RUNTIME;
+     mysql> SAVE MYSQL USERS TO DISK;
+
+  To confirm that the user has been set up correctly, you can try to log in:
+
+  .. code-block:: bash
+
+     root@proxysql:~# mysql -u sbuser -psbpass -h 127.0.0.1 -P 6033
+
+     Welcome to the MySQL monitor.  Commands end with ; or \g.
+     Your MySQL connection id is 1491
+     Server version: 5.5.30 (ProxySQL)
+
+     Copyright (c) 2009-2020 Percona LLC and/or its affiliates
+     Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+     Oracle is a registered trademark of Oracle Corporation and/or its
+     affiliates. Other names may be trademarks of their respective
+     owners.
+
+     Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+  To provide read/write access to the cluster for ProxySQL,
+  add this user on one of the |PXC| nodes:
+
+  .. code-block:: text
+
+     mysql> CREATE USER 'sbuser'@'192.168.70.64' IDENTIFIED BY 'sbpass';
+     Query OK, 0 rows affected (0.01 sec)
```

Comment:
We should create user with mysql_native_password plugin: 
https://github.com/sysown/proxysql/issues/2580

Reply:
@borys-belinsky Ignore this for v5.7. Default password plugin is mysql_native_password in v5.7